dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINSubModel.h

```diff
@@ -37,6 +37,7 @@
 
 namespace DYN {
 class SubModel;
+struct LocalInitParameters;
```

Comment:
This introduces a dependency between solvers and models which is strictly forbidden. Please use a parameterSet for example

Reply:
There is a dependency between solvers and models already (l.39 "class SubModel;").